setup.py

```diff
@@ -38,11 +35,6 @@ def run(self):
         _sdist.run(self)
 
 
-class install(_install):
```

Comment:
fwiw, this is only complicated because of these weird submodule dependency on `uap-core`. :)